include/envoy/server/admin.h

```diff
@@ -33,7 +36,9 @@ class Admin {
    * @param response supplies the buffer to fill in with the response body.
    * @return Http::Code the response code.
    */
-  typedef std::function<Http::Code(const std::string& url, Buffer::Instance& response)> HandlerCb;
+  typedef std::function<Http::Code(const std::string& url, Http::HeaderMap& header,
```

Comment:
Can you add this to the @param list above?

Reply:
done

Reply:
nit: I think this one should be `reponse_headers` also.

Reply:
done